src/charts/chartOptions.html

```diff
@@ -119,7 +119,7 @@
             <div class="cell" rv-show="showShare">
                 <div class="chartOptions_button" rv-on-click="toggleExportSelector" rv-class-button_selected="showExportSelector">
                     <span class = "shareButton" data-balloon="Share chart" data-balloon-pos="down">
-                        <div class="img"><img src="images/share.svg"></div>
+                        <div class="img"><img src='data:image/png;base64,R0lGODlhFAAUAIAAAP///wAAACH5BAEAAAAALAAAAAAUABQAAAIRhI+py+0Po5y02ouz3rz7rxUAOw==' class="share-icon"></div>
```

Comment:
@4p00rv You added image in the HTML file. It will not go through out image optimization grunt command.